web/src/lib/components/ListView.svelte

```diff
@@ -4,6 +4,7 @@
     import { token } from "$lib/stores";
     import type { IEntity } from '$lib/types/base';
     import Loading from "$lib/components/Loading.svelte";
+    import MetaTag from './MetaTag.svelte';
```

Comment:
let's use absolute imports, starting from "$lib/" - I know it is not that way everywhere yet, but I think it is cleaner and we should transition to that.